src/Sarif/Autogenerated/ResourcesEqualityComparer.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.CodeDom.Compiler;
 using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Sarif;
```

Comment:
>Sarif [](start = 29, length = 5)

Odd. Was this a hand-edit you forgot to revert? I don't remember making a code gen change that would have had the effect of generating this superfluos `using` directive.